packages/cw-orch-mock/src/core.rs

```diff
@@ -298,6 +310,15 @@ impl<S: StateInterface> TxHandler for Mock<S> {
     fn block_info(&self) -> Result<cosmwasm_std::BlockInfo, CwEnvError> {
         Ok(self.app.borrow().block_info())
     }
+
+    /// Specify Empty for the type of R if you don't know the type to unwrap the response to
+    fn commit_any<R>(
+        &self,
+        _msgs: Vec<cosmrs::Any>,
+        _memo: Option<&str>,
+    ) -> Result<Self::Response, Self::Error> {
+        panic!("Un-implemented, mock env doesn't support starship messages yet")
```

Comment:
`starship` -> `stargate`

I wonder if it makes sense to have this in the `TxHandler` trait because it isn't implementable in `Mock`. 

If a user writes a scripts with a generic `Chain: CwEnv`, he expects the script to work with any type that meets that requirement. With this addition that is no longer the case which is not good imo :/ 


Can it be implemented maybe through a `Stargate` trait or something similar that is only implemented for Daemon and Test-Tube for now?

Reply:
Yes good idea ! I even created an additional `cw-orch-traits` package for keeping the core stable and having additional traits implemented as we go if needed (Bank, Staking or other tx and query capabilites)